includes/class.llms.review.php

```diff
@@ -40,64 +41,110 @@ public function __construct() {
+
 			?>
 			<div id="old_reviews">
-			<h3><?php echo apply_filters( 'lifterlms_reviews_section_title', __( 'What Others Have Said', 'lifterlms' ) ); ?></h3>
-			<?php
-			$args        = array(
-				'posts_per_page'   => get_post_meta( get_the_ID(), '_llms_num_reviews', true ),
-				'post_type'        => 'llms_review',
-				'post_status'      => 'publish',
-				'post_parent'      => get_the_ID(),
-				'suppress_filters' => true,
-			);
-			$posts_array = get_posts( $args );
+				<h3><?php echo esc_html( $section_title ); ?></h3>
+				<?php
+				$args = array(
+					'posts_per_page'   => get_post_meta( get_the_ID(), '_llms_num_reviews', true ),
+					'post_type'        => 'llms_review',
+					'post_status'      => 'publish',
+					'post_parent'      => get_the_ID(),
+					'suppress_filters' => true,
+				);
 
-			$styles = array(
-				'background-color' => '#EFEFEF',
-				'title-color'      => 'inherit',
-				'text-color'       => 'inherit',
-				'custom-css'       => '',
-			);
+				$posts_array = get_posts( $args );
 
-			if ( has_filter( 'llms_review_custom_styles' ) ) {
-				$styles = apply_filters( 'llms_review_custom_styles', $styles );
-			}
+				/**
+				 * Allow review custom styles to be filtered.
+				 *
+				 * @since 1.2.7
+				 *
+				 * @param array $styles Array of custom styles.
+				 */
+				$styles = apply_filters(
+					'llms_review_custom_styles',
+					array(
+						'background-color' => '#efefef',
+						'title-color'      => 'inherit',
+						'text-color'       => 'inherit',
+						'custom-css'       => '',
+					)
+				);
 
-			foreach ( $posts_array as $post ) {
-				echo $styles['custom-css'];
+				$inline_styles = '';
 
+				if ( $styles['background-color'] ?? '' ) {
+					$inline_styles .= '.llms_review{background-color:' . $styles['background-color'] . '}';
+				}
+
+				if ( $styles['title-color'] ?? '' ) {
+					$inline_styles .= '.llms_review h5{color:' . $styles['title-color'] . '}';
+				}
+
+				if ( $styles['text-color'] ?? '' ) {
+					$inline_styles .= '.llms_review h6,.llms_review p{color:' . $styles['text-color'] . '}';
+				}
+
+				if ( $styles['custom-css'] ?? '' ) {
+
+					// Remove style tags in case they were added with the filter.
+					$inline_styles .= str_replace( array( '<style>', '</style>' ), '', $styles['custom-css'] );
+				}
+
+				if ( $inline_styles ) {
+					echo '<style id="llms_review_custom_styles">' . $inline_styles . '</style>';
+				}
+
+				foreach ( $posts_array as $post ) {
+					?>
+					<div class="llms_review">
+						<h5><strong><?php echo get_the_title( $post->ID ); ?></strong></h5>
+						<h6>
+							<?php
+							// Translators: %s = The author display name.
+							echo esc_html( sprintf( __( 'By: %s', 'lifterlms' ), get_the_author_meta( 'display_name', get_post_field( 'post_author', $post->ID ) ) ) );
+							?>
+						</h6>
+						<p><?php echo esc_html( get_post_field( 'post_content', $post->ID ) ); ?></p>
```

Comment:
mmm `get_post_field()` should sanitize the field already, can you check we need the `esc_html()` here?

Reply:
@eri-trabiccolo I think you're right, `get_post_field()` does sanitize the value